prelude/bootstrap.js

```diff
@@ -1630,6 +1631,13 @@ function payloadFileSync(pointer) {
     const opts = args[pos];
     if (!opts.env) opts.env = require('util')._extend({}, process.env);
     if (opts.env.PKG_EXECPATH === 'PKG_INVOKE_NODEJS') return;
+    if (Array.isArray(opts.execArgv)) {
+      // fork: The patched Node.js binary prevents us from passing execArgv on
+      // the cli. Instead we'll pass those options through the environment.
+      // https://nodejs.org/docs/latest/api/cli.html#cli_node_options_options
+      opts.env.NODE_OPTIONS = opts.execArgv.join(' ');
+      delete opts.execArgv;
+    }
```

Comment:
First, I know this is a workaround so I don't know how welcome that kind of change is.

Second, would it make sense to move that logic right above the `PKG_INVOKE_NODEJS` check?